plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/DeployPropertyPage.java

```diff
@@ -69,18 +72,20 @@ protected Control createContents(Composite parent) {
       return container;
```

Comment:
I've updated the PR that it will show the `BlankDeployPreferencesPanel` however not by explicitly returning it here.